int32[] seg_idxs
float64[] ts
std_msgs/Float64MultiArray[] xs
std_msgs/Float64MultiArray[] us